core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -486,6 +486,11 @@ public class GeyserSession implements GeyserConnection, GeyserCommandSource {
     @Setter
     private float walkSpeed;
 
+    @Setter
+    private boolean requestedPacks = false;
```

Comment:
This imo is a bit misleading - a client could have packs without requesting them from the server. IMO this shouldn't be added, as only a geyser optional pack detection would be something of direct use

Reply:
removed in latest commit